src/foam/u2/Element.js

```diff
@@ -1320,7 +1320,13 @@ foam.CLASS({
       return this.entity('nbsp');
     },
 
-    function addClass(cls) { /* Slot | String */
+    function addClass(cls) { /* Slot | String | Array[String, Slot] */
```

Comment:
I think this comment is wrong because you aren't passing in an Array. Better:
/* ( Slot | String )... */